mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.erb

```diff
@@ -1254,6 +1259,14 @@ func flattenBucketLifecycleRuleCondition(condition *storage.BucketLifecycleRuleC
 			ruleCondition["with_state"] = "ARCHIVED"
 		}
 	}
+  // setting no_age value from state config since it is terraform only variable and not getting value from backend.
```

Comment:
Nit: some more spacing inconsistencies in this block. It makes it a little hard to read.